### ISSUE TITLE
Есть ли такой скрипт для Pfsence ?

### ISSUE THREAD

**Issue by grashooper** [Limych/antizapret#6]
Нет некоторых настроек в Pfsence, которые описаны. Напирмер Type = External (advanced)\
Следовательно невозможно использовать. 
Так же попробовал установить git и запустить скрипт.
Вот что выдала консоль 
sh opnsense/install.sh
ln: /usr/local/opnsense/service/conf/actions.d/actions_antizapret.conf: No such file or directory
configd does not exist in /etc/rc.d or the local startup
directories (/usr/local/etc/rc.d), or is not executable

В общем нужен другой способ для Pfsence

**Comment by Limych** [comment]
Я никогда не работал с pfsense, потому не могу ничего гарантировать, извините.

Скрипт-инсталлятор, да, только для OPNsense. Он у вас и не должен работать.
Если мы с Вами выработаем точный алгоритм для Вашей системы, добавлю второй.

Какие варианты настройки у Вас есть? Можно ли подгрузить список из локального файла? А с веб-сайта?

**Comment by grashooper** [comment]
У меня настроен TOR, но он работает по портам HTTP и HTTPS соответственно на хороших скоростях. Это очень не удобно так как весь трафик этих портов пускается через TOR.
Тор качал из репозитория FreeBSD
Думаю с локального можно сделать.
Так же работает и сокс 5-й, но там скорость совсем плохая.
Раньше ставил скриптом на роутер с прошивкой от Padavan. Хороший был скрипт от Blackcoffe. но он удалил все свои идеи, скрипты и канал на YouTube. Я сохранил его скрипт для роутера на прошивке Падавана, но этот скрипт теперь перестал работать.
Так вот, по сути тор уже настроен и работает на Pfsense. Теперь надо как-то привязать определённый список IP и доменов из списка антизапрета чтобы только эти адреса направлялись на TOR
Может какие идеи будут из того скрипта https://github.com/grashooper/rublock-tor

В алиасах Pfsense есть по портам и IP, а так же хосты и таблицы IP и портов.

**Comment by grashooper** [comment]
Да, ещё. Если интересно, то вот сделал статью как я настраивал TOR на Pfsense
Там подробно описал всё по пунктам и с фотографиями. Я не видел подобной настройки нигде. 
https://zen.yandex.ru/media/id/61af14da337f233ae51d837b/pfsense-tor-shliuz--proksi-61af14f3d0e137381ee89e6f
